precompile/contracts/rewardmanager/contract_test.go

```diff
@@ -128,17 +127,19 @@ func TestRewardManagerRun(t *testing.T) {
 			ReadOnly:    false,
 			ExpectedRes: func() []byte {
 				res, err := PackCurrentRewardAddressOutput(testAddr)
-				require.NoError(t, err)
+				if err != nil {
+					panic(err)
+				}
```

Comment:
curious why we use `panic` here instead of `t.Fatal`

Reply:
No `t` is available here anymore and it should never error here for a correctly written test - a cleaner way to do this would be to construct the needed byte array before these tests